packages/alfa-rules/src/sia-er62/rule.ts

```diff
@@ -0,0 +1,614 @@
+  ).value;
+
+  return hasComputedStyle(
+    "vertical-align",
+    not((alignment) => alignment.equals(reference)),
+    device,
+    context
+  );
+}
+
+function hasDistinguishableCursor(
+  container: Element,
+  device: Device,
+  context?: Context
+): Predicate<Element> {
+  // Checking if there is a custom cursor, otherwise grabbing the built-in
+  function getFirstCursor(style: Style.Computed<"cursor">) {
+    return style.values[0].values.length !== 0
+      ? style.values[0].values[0]
+      : style.values[1];
+  }
+  const containerCursorStyle = Style.from(container, device, context).computed(
+    "cursor"
+  ).value;
+
+  // We assume that the first custom cursor, if any, will never fail to load
+  // and thus don't try to default further.
+  const reference = getFirstCursor(containerCursorStyle);
+
+  return hasComputedStyle(
+    "cursor",
+    not((cursor) => getFirstCursor(cursor).equals(reference)),
+    device,
+    context
+  );
+}
+
+type Name = Property.Name | Property.Shorthand.Name;
+
+export class ComputedStyles implements Equatable, Hashable, Serializable {
+  public static of(
+    style: Iterable<readonly [Name, string]> = []
+  ): ComputedStyles {
+    return new ComputedStyles(Map.from(style));
+  }
+
+  private readonly _style: Map<Name, string>;
+
+  private constructor(style: Map<Name, string>) {
+    this._style = style;
+  }
+
+  public get style(): Map<Name, string> {
+    return this._style;
+  }
+
+  public with(
+    ...styles: ReadonlyArray<readonly [Name, string]>
+  ): ComputedStyles {
+    return ComputedStyles.of([...this._style, ...styles]);
+  }
+
+  public equals(value: ComputedStyles): boolean;
+
+  public equals(value: unknown): value is this;
+
+  public equals(value: unknown): boolean {
+    return value instanceof ComputedStyles && value._style.equals(this._style);
+  }
+
+  public hash(hash: Hash): void {
+    this._style.hash(hash);
+  }
+
+  public toJSON(): ComputedStyles.JSON {
+    return {
+      style: this._style.toJSON(),
+    };
+  }
+}
+
+export namespace ComputedStyles {
+  export interface JSON {
+    [key: string]: json.JSON;
+    style: Map.JSON<Name, string>;
+  }
+
+  export function isComputedStyles(value: unknown): value is ComputedStyles {
+    return value instanceof ComputedStyles;
+  }
+
+  export function from(
+    element: Element,
+    device: Device,
+    target: Element,
+    context: Context = Context.empty()
+  ): ComputedStyles {
+    const style = Style.from(element, device, context);
+
+    const border = (["color", "style", "width"] as const).map((property) =>
+      Serialise.borderShorthand(style, property)
+    );
+
+    const cursor = context.isHovered(target)
+      ? [["cursor", Serialise.getLonghand(style, "cursor")] as const]
+      : [];
+
+    return ComputedStyles.of(
+      [
+        ...border,
+        ...cursor,
+        ["color", Serialise.getLonghand(style, "color")] as const,
+        ["font", Serialise.font(style)] as const,
```

Comment:
Do the `font` and `cursor` changes that this PR covers make [this PR](https://github.com/Siteimprove/alfa/pull/968/files) obsolete? If yes, should we close that PR when this is merged?